examples/window_debug.rs

```diff
@@ -85,10 +86,17 @@ fn main() {
                         if window.fullscreen().is_some() {
                             window.set_fullscreen(None);
                         } else {
-                            let monitor = window.current_monitor().unwrap();
+                            let monitor = window.current_monitor();
                             window.set_fullscreen(Some(Fullscreen::Borderless(monitor)));
```

Comment:
Isn't this the same as Borderless(None)?

Reply:
Well, yeah, but that's purely for testing. Could be different on Wayland in certain cases though.